doc/classes/AnimationTreePlayer.xml

```diff
@@ -610,6 +619,16 @@
 	</methods>
 	<members>
 		<member name="playback_process_mode" type="int" setter="set_animation_process_mode" getter="get_animation_process_mode" enum="AnimationTreePlayer.AnimationProcessMode">
+			The process notification in which to update animations. Default value: [enum ANIMATION_PROCESS_IDLE].
+		</member>
+		<member name="master_player" type="NodePath" setter="set_master_player" getter="get_master_player">
+			The [AnimationPlayer] node responsible for controlling this tree's animations.
+		</member>
+		<member name="base_path" type="NodePath" setter="set_base_path" getter="get_base_path">
+			The node from which to relatively access other nodes. Default value: [code]".."[/code].
+		</member>
+		<member name="active" type="bool" setter="set_active" getter="is_active">
+			If [code]true[/code] updates animations in response to process-related notifications. Default value: [code]false[/code].
```

Comment:
I imagine some users don't know about NOTIFICATION_[PHYSICS]_PROCESS and they are just using 'sugar' functions.  
Maybe something simpler like in AnimationPlayer:
"Set the player as active (playing). If false, it will do nothing."
so it would be:
"Set the AnimationTreePlayer as active (playing). If false, it will do nothing."

Reply:
Well, it's weird because being active doesn't necessarily mean that it's playing. It means that it is ABLE to play, because it is picking up on the notifications, but without it, you aren't hooking into the delta times at all and therefore won't update animations even if you are supposed to be "playing" them.

Maybe...
`If [code]true[/code] the AnimationTreePlayer is able to play animations.`

Reply:
I have a separate PR that updated the Animation API docs to reflect that information already.